_posts/2019-05-10-server-rendering-responsively.md

```diff
@@ -0,0 +1,189 @@
+---
+layout: epic
+title: "Server-Rendering Responsively"
+date: "2019-04-22"
+author: [steve-hicks]
+categories: [react, html, web]
+---
+
+We use server-side rendering (SSR) to deliver every page you hit on [artsy.net](https://artsy.net). We decided on
+using SSR for many reasons, amongst them performance. (TODO: Maybe more about why we chose SSR, but I don't know
+much about that history.)
```

Comment:
Here's some interesting history, as well as some whys: https://artsy.github.io/blog/2013/11/30/rendering-on-the-server-and-client-in-node-dot-js/. Artsy was amongst the first to come up with a good pattern.

Reply:
Oooo, I like this - I will link to it, and I think that's the little extra kick I was looking for in this paragraph.